azurerm/resource_arm_redis_cache.go

```diff
@@ -101,6 +102,28 @@ func resourceArmRedisCache() *schema.Resource {
 							Default:      "volatile-lru",
 							ValidateFunc: validateRedisMaxMemoryPolicy,
 						},
+						"rdb_backup_enabled": {
+							Type:             schema.TypeString,
+							Optional:         true,
+							DiffSuppressFunc: ignoreCaseDiffSuppressFunc,
+							ValidateFunc: validation.StringInSlice([]string{
```

Comment:
this was originally due to the API design - but by chatting in Slack we've managed to work around this 👍